taskwiki/viewport.py

```diff
@@ -29,7 +29,7 @@ class ViewPort(object):
           * [ ] Make sure the hosting is working
     """
 
-    meta_tokens = ("-VISIBLE",)
+    meta_tokens = ('-VISIBLE',)
```

Comment:
This commit can be rebased and squashed into the first one.

Reply:
I will squash (or you can via he github interface) everything into a single commit, no need to pollute the log.

Reply:
I just squashed everything into the first commit.